clouddriver-openstack/src/main/resources/asg_server_resource.yaml

```diff
@@ -0,0 +1,41 @@
+heat_template_version: 2016-04-08
+description: An auto-scaled server for Spinnaker without any load balancer association.
```

Comment:
No ... We have embedded templates, one for an asg or server group, and two other ones each to describe the resource that is being auto-scaled.  One of the resources uses lbaasv2 to associated to a load balancer, the other is simply a server resource.